proxy/proxies/nginx/nginx.go

```diff
@@ -50,10 +52,12 @@ import (
 )
 
 const (
-	defaultNginxIngressImage = "cargo.caicloud.io/caicloud/ingress-nginx:v0.1.0"
+	defaultNginxIngressImage = "cargo.caicloud.io/caicloud/nginx-ingress-controller:0.9.0-beta.8"
```

Comment:
I know user can specify the image path. But I don't think you should use a default image. In some cases, the path `cargo.caicloud.io/*` and `cargo.caicloudprivatetest.com/*` can't be accessed. 

The option should not be optional.